### ISSUE TITLE
Add eu-central-1 AMI images

### ISSUE THREAD

**Issue by skurfuerst** [amplab/spark-ec2#13]
Hey everybody,

due to company policy restrictions, we are just allowed to use the eu-central-1 AWS region -- unfortunately, the Spark-EC2 image is not added there...

Would be really great for me if you could add it!

All the best,
Sebastian

**Comment by cederigo** [comment]
:+1:

**Comment by andkopp** [comment]
I would appreciate.

**Comment by nchammas** [comment]
@skurfuerst - Are you able to manually copy the spark-ec2 AMI to the region you need?

I might be wrong about this, but I don't think the Spark team has the resources to spend on adding and managing more custom spark-ec2 AMIs. The ones we already have are quite old and haven't been updated in years.

**Comment by shivaram** [comment]
Yeah I'd just like to add that maintaining / updating AMIs regularly is something we don't have manpower for. I think we will do a new set for Spark 2.0 and in that batch we will try to include eu-central-1 as well.
